google-cloud/README.md

```diff
@@ -78,20 +99,32 @@ We'll use the Houston Python client to save the plan.
    export HOUSTON_KEY='<your api key>'
    ```
    **Never commit this key anywhere in your repo!**
+   
+   If you have a self-hosted solution then you will need another environment variable for the URL and port for your 
+server.
+   ```bash
+   export HOUSTON_URL='<your Houston url>'
+   ```
+   
+    If you didn't use the default value for `houston_key_name` then another environment variable is needed to specify 
+that key name.
+    ```bash
+    export HOUSTON_KEY_SECRET_NAME='<your Houston key secret name>'
```

Comment:
We should use the default name for the quickstart. Please remove this step

Reply:
Agreeded on adding an appendix in Google DMs

Reply:
Appendix has been created and references to key secret name moved to it